packages/react-native-editor/__device-tests__/pages/editor-page.js

```diff
@@ -553,6 +580,46 @@ class EditorPage {
 	async sauceJobStatus( allPassed ) {
 		await this.driver.sauceJobStatus( allPassed );
 	}
+
+	// =============================
+	// Misc functions
+	// =============================
+	async findElement( name, iosElementType ) {
```

Comment:
Running xpath locator `elementsByXPath` with `*[contains` could be slow and is [not recommended by Appium because of performance issues](https://appium.io/docs/en/commands/element/find-elements/index.html#selector-strategies). Instead we can use `elementByAccessibilityId`.
If that's not possible in this case, maybe we can rename the function to something like `findElementByXPath`.

Reply:
Unfortunately, I made this function because I couldn't for the life of me find the element by accessibility id. I can rename the function.